### PR TITLE
Add typescript paragraph

### PR DESCRIPTION
**Pull request by AlexGalays**

### CODE REVIEW

**Comment by briancavalier** [comment]
Thanks @AlexGalays! I'll have time to look at this later today.

File: README.md [hunk 1]
```diff
@@ -106,6 +106,13 @@ bower install --save most`
 <script src="most/dist/most.js"></script>
 ```
 
+### Typescript support
+
+Most.js works with typescript out of the box as it provides local typings that will be read when you import Most.js in your code. You do not need to manually link an external `d.ts` file in your tsconfig.
```
Comment (briancavalier) [inline README.md]: How about linking "provides local typings" to https://github.com/cujojs/most/blob/master/type-definitions/most.d.ts

File: README.md [hunk 2]
```diff
@@ -106,6 +106,13 @@ bower install --save most`
 <script src="most/dist/most.js"></script>
 ```
 
+### Typescript support
+
+Most.js works with typescript out of the box as it provides local typings that will be read when you import Most.js in your code. You do not need to manually link an external `d.ts` file in your tsconfig.  
+Most.js has a dependency on native Promises so a type definition for Promise must be available in your setup:
```
Comment (briancavalier) [inline README.md]: It seems like there should either be another line break before this sentence to make sure it gets formatted as a new paragraph in markdown, or it should be combined with the previous paragraph.

**Comment by briancavalier** [comment]
Looks great.  Just a couple minor comments.

**Comment by AlexGalays** [comment]
Updated

**Comment by briancavalier** [comment]
Thanks!

**Pull request merged**